components/image/slider/src/index.js

```diff
@@ -0,0 +1,60 @@
+import React, {PropTypes} from 'react'
+import ReactSlidy from 'react-slidy'
+import SuiMultimedia from '@schibstedspain/sui-multimedia'
```

Comment:
Here we're using a deprecated component, I suggest to use this one https://github.com/SUI-Components/sui-components/tree/master/components/image/lazyLoad